contributors/devel/architectural-roadmap.md

```diff
@@ -415,7 +415,7 @@ The Controller Manager and Kubelet currently call out to a "cloud
 provider" interface to query information from the infrastructure layer
 and/or to manage infrastructure resources. However, [we’re working to
 extract those
-touchpoints](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cloud-provider-refactoring.md)
+touchpoints](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cloud-provider/cloud-provider-refactoring.md)
```

Comment:
same repo. This can be:
`touchpoints](/contributors/design-proposals/cloud-provider/cloud-provider-refactoring.md)`